frontend/src/components/VSearchResultsGrid/VImageCell.vue

```diff
@@ -130,12 +132,19 @@ export default defineComponent({
       imgWidth.value = element.naturalWidth
     }
 
+    const contextSensitiveTitle = computed(() => {
+      return i18n.t("browse-page.aria.image-title", {
+        title: props.image.title,
```

Comment:
Do you think it would be clearer for screen reader users if we added more info, something like the full attribution sentence (Image: Cat by creator licensed under CC BY licensed, maybe even adding the provider)?

Reply:
I support this idea. Perhaps it's better-suited to a follow-up PR though, as I'd want to make sure, and check with a11y folks, that the titles wouldn't be too long and hinder the search browsing experience.